src/python/pants/bin/pants_loader.py

```diff
@@ -37,14 +38,17 @@ def ensure_locale(cls):
     # libraries called by Pants may fail with more obscure errors.
     encoding = locale.getpreferredencoding()
     if encoding.lower() != 'utf-8' and os.environ.get(cls.ENCODING_IGNORE_ENV_VAR, None) is None:
-      raise cls.InvalidLocaleError(
-        'System preferred encoding is `{}`, but `UTF-8` is required.\n'
-        'Check and set the LC_* and LANG environment settings. Example:\n'
-        '  LC_ALL=en_US.UTF-8\n'
-        '  LANG=en_US.UTF-8\n'
-        'To bypass this error, please file an issue and then set:\n'
-        '  {}=1'.format(encoding, cls.ENCODING_IGNORE_ENV_VAR)
-      )
+      raise cls.InvalidLocaleError(dedent("""\
+        Your system's preferred encoding is `{}`, but Pants requires `UTF-8`.
+        Check and set the LC_* and LANG environment settings. Example:
+          LC_ALL=en_US.UTF-8
+          LANG=en_US.UTF-8
+        Specifically, Python's `locale.getpreferredencoding()` must resolve to `UTF-8`.
```

Comment:
Does it make more sense for this sentence to be moved above the example, to be the second sentence in paragraph?